src/pip/_internal/commands/install.py

```diff
@@ -156,7 +156,12 @@ def add_options(self) -> None:
             default=None,
             help=(
                 "Installation prefix where lib, bin and other top-level "
-                "folders are placed"
+                "folders are placed. \n"
+                "Note that the resulting installation may "
+                "contain scripts and other resources which reference the "
+                "Python interpreter of pip, and not that of ``--prefix``. \n"
+                "See also the ``pip --python`` option if the intention is to "
+                "install packages into another (possibly pip-free) environment."
```

Comment:
I'm not sure that `\n` render correctly, can you check? Otherwise sounds good to me.

Reply:
You're right. They get scrubbed in the help, and get scrubbed in the HTML. I'll remove them.